packages/validator/docs/actions.md

```diff
@@ -0,0 +1,292 @@
+# Actions
```

Comment:
This should be definitely done, but its a bit tricky, I will have to check what options do we have and it might take a bit time, so I created [new issue](https://github.com/api3dao/airnode/issues/423) specifically for this.

Reply:
Yeah there shouldnt be many onconsistencies, they can stay for now. I will look into "autodocumentation" right after we get this merged.
> Also, will the validator README be part of API3 docs?
Good question, I will bring it up on the next dev call.